zappnode-front/src/landing/components/header/navigation.js

```diff
@@ -2,41 +2,60 @@ import React, {useState, useEffect} from 'react';
 import '../css/tailwind.generated.css';
 import '../css/landing.css';
 
+// navigation component 
+// please read and understand the code before making changes
+// the component is build on tailwind css so incase you see wired classes
 const Navigation = props => {
+  
+
+  const handleClick = (e) => {
+    e.preventDefault();
+    if(!props.toogle){
+     props.click(true)
+    }else{
+      props.click(false)
+    }
+
+    console.log(props.toogle)
+  }
+
+  
+  useEffect(()=> {
+    console.log('effect fired')
+  }, [handleClick]);
+
```

Comment:
ESLint: The 'handleClick' function makes the dependencies of useEffect Hook (at line 25) change on every render.

Reply:
yeah, am using it to work on navigation when am done, I will get rid of it